de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/concept/types/ABeanObject.java

```diff
@@ -39,11 +49,17 @@ public ABeanObject() {
 	}
 	
 	@SuppressWarnings("unchecked")
+	@XmlID
+	@XmlElement(name = "uuid")
+	@XmlJavaTypeAdapter(TypeInstanceAdapter.class)

```

Comment:
I know Im the one who added this :D but the way I understand it now we only neet to annotate the set or the get.
I see in the other methods you annotate the get, so to stay consistent, I think here too we should only annotate the getter.

Reply:
Sure and I also found some other beans where I did the same